dashboard/common/db_util.py

```diff
@@ -85,7 +84,7 @@ def get_default_user_course_id(user_id):
     return course_id
 
 
-def get_user_courses_info(username: str) -> List[Dict[str, Union[str, int, List[str]]]]:
+def get_user_courses_info(request, username: str) -> List[Dict[str, Union[str, int, List[str]]]]:
```

Comment:
Add a type hint? I think it's [`HttpRequest`](https://docs.djangoproject.com/en/3.1/ref/request-response/#httprequest-objects), but a log message would confirm.

Reply:
It is a `django.core.handlers.wsgi.WSGIRequest` request

Reply:
Shows what I know (not much)!

Reply:
Added the typehint with WSGIRequest